src/Microsoft.ML.SearchSpace/Tuner/GridSearchTuner.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.ML.SearchSpace.Tuner
+{
+    public class GridSearchTuner
+    {
+        private readonly SearchSpace _searchSpace;
+        private readonly int _stepSize;
+
+        public GridSearchTuner(SearchSpace ss, int stepSize = 10)
```

Comment:
`ss` => `searchSpace`. Don't use abbreviations in public API.

Reply:
### Resolved